src/main/java/com/target/ready/library/system/Controller/LibraryController.java

```diff
@@ -18,10 +18,20 @@ public class LibraryController {
     LibraryController(LibrarySystemService librarySystemService){
         this.librarySystemService=librarySystemService;
     }
-    @GetMapping("getBooks")
-    public List<Book> getAllBooks(){
-
-        return librarySystemService.getAllBooks();
+    @GetMapping("books_directory")
+    public List<Book> getAllBooks(@RequestParam(value = "pageNumber", defaultValue = "0", required = false) Integer pageNumber){
```

Comment:
Request params should also be in snake case.

Reply:
so should i change pageNumber to page_number??

Reply:
@PostMapping("/api/foos")
@ResponseBody
public String addFoo(@RequestParam(name = "id") String fooId, @RequestParam String name) { 
    return "ID: " + fooId + " Name: " + name;
}    i found this in the documentation https://www.baeldung.com/spring-request-param where Request Params are in camel case so accordingly i used a camel case